src/ralph/converters/xapi_converter.py

```diff
@@ -0,0 +1,67 @@
+"""Entrypoint to convert events to xAPI"""
+
+import json
+import logging
+from enum import Enum
+
+from .xapi.base import BaseXapiConverter
+from .xapi.server import ServerXapiConverter
+
+# converters module logger
+logger = logging.getLogger(__name__)
+
+
+class Converters(Enum):
+    """Stores initialized xAPI converters"""
+
+    SERVER = ServerXapiConverter()
+
+
+class XapiConverter:
+    """Uses xapi converters to convert to xAPI format"""
+
+    def __init__(self, event, platform):
+        """Initialise Xapi Converter
+
+        Args:
+            event (str): event to convert
+            platform (str): url of the platform to which the event belongs
+        """
+        BaseXapiConverter._platform = platform
```

Comment:
I don't know why but pylint don't complain about protected-access here. Any Ideas ? May be we could keep it this way or it's better to extract this in a public function ?